data/tlg0018/tlg001/tlg0018.tlg001.1st1K-eng1.xml

```diff
@@ -0,0 +1,1013 @@
+</div>
+
+<div type="textpart" subtype="section" n="34">
+<p>and these boundaries are evening and morning; the one of which heralds in the good tidings that the sun is about to rise, gently dissipating the darkness: and evening comes on as the sun sets, receiving gently the collective approach of darkness. And these, I mean morning and evening, must be placed in the class of incorporeal things, perceptible only by the intellect; for there is absolutely nothing in them which is perceptible by
+<pb n="9"/>
+the external senses, but they are entirely ideas, and measures, and forms, and seals, incorporeal as far as regards the generation of other bodies.</p>
+</div>
+
+<div type="textpart" subtype="section" n="35">
+<p>But when light came, and darkness retreated and yielded to it, and boundaries were set in the space between the two, namely, evening and morning, then of necessity the measure of time was immediately perfected, which also the Creator <choice><sic>called "day,"</sic><corr>called "day."</corr></choice> and He called it <corr>not "the</corr> first day," <corr>but "one</corr> day;" and it is spoken of thus, on account of the single nature of the world perceptible only by the intellect, which has a single nature.
+<milestone unit="chapter" n="10"/></p>
+</div>
+
+<div type="textpart" subtype="section" n="36">
+<p>The incorporeal world then was already completed, having its seat in the Divine Reason; and the world, perceptible by the external senses, was made on the model of it; and the first portion of it, being also the most excellent of all made by the Creator, was the heaven, which he truly called the firmament, as being corporeal; for the body is by nature firm, inasmuch as it is divisible into three parts; and what other idea of solidity and of body can there be, except that it is something which may be measured in every direction? therefore he, very naturally contrasting that which was perceptible to the external senses, and corporeal with that which was perceptible only by the intellect and incorporeal, called this the firmament.</p>
+</div>
+
+<div type="textpart" subtype="section" n="37">
+<p>Immediately afterwards he, with great propriety and entire correctness, called it the heaven, either because it was already the boundary <ref n="9.1"/> of everything, or because it was the first of all visible things which was created; and after its second rising he called the time day, referring the entire space and measure of a day to the heaven, on account of its dignity and honour among the things perceptible to the external senses.
+<milestone unit="chapter" n="11"/></p>
```

Comment:
see above

Reply:
So that fixed three of the files, it also turns out that tlg0062.tlg018.1st1K-grc1.xml had duplicated nodes, it has two section 34s. So I'm going tofix that next and see if that at least makes all the files valid.